test/e2e/minscale_readiness_test.go

```diff
@@ -126,11 +131,21 @@ func TestMinScale(t *testing.T) {
 		t.Fatalf("The revision %q observed scale %d < %d after becoming ready", newRevName, lr, minScale)
 	}
 
+	revision, err = clients.ServingClient.Revisions.Get(context.Background(), newRevName, metav1.GetOptions{})
```

Comment:
usually you want two if statements: one for `err` and one for `value` otherwise the failure messages are kind of hard to read.